packages/web-react/src/components/Accordion/Accordion.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import { ComponentMeta } from '@storybook/react';
+import Accordion from './Accordion';
+
+export default {
+  title: 'Components/Accordion',
+  parameters: {
+    docs: {
+      description: {
+        component: `
+  [Spirit Accordion](https://spirit.supernova-docs.io/spirit/latest/components/accordion/overview.html)
+
+  Toggle the visibility of content across your project with Accordion.
+  `,
+      },
+    },
+  },
+} as ComponentMeta<typeof Accordion>;
+
+export { default as Accordion } from './stories/Accordion';
+export { default as AccordionOneOpened } from './stories/AccordionOneOpened';
```

Comment:
I think that we should switch these two examples. Because only one open is the default behavior with a simple toggle function.

The second example is the `StayOpen` one. I think that in this case, it makes sense to be the same as Reacstrap.

Reply:
done